mysql-test/main/status.result

```diff
@@ -447,3 +447,22 @@ Feature_json	2
 connection default;
 set @@global.concurrent_insert= @old_concurrent_insert;
 SET GLOBAL log_output = @old_log_output;
+#
+# MDEV-30543 New status variable: Max_used_connections_time
+#
+FLUSH STATUS;
+connect  con1,localhost,root,,;
+connect  con2,localhost,root,,;
+connection con1;
+disconnect con2;
+SELECT 'DTVALUE' = 'DTVALUE' AS expect_1;
+expect_1
+1
```

Comment:
Great! Much easier to interpret both the `.result` file and the `.test` file in this form.